api-docs/get-bta-tile-data.md

```diff
@@ -106,20 +106,20 @@ Calls to this API must be made by an authenticated and authorised user with an A
 
 Same payloads as approved but the status will be "Insolvent"
 
-***De-registered / Revoked / Small Cider Producer:***
+***Deregistered / Revoked / Small Cider Producer:***
 
 ```json
 {
   "alcoholDutyReference": "AP0000000001", 
-  "approvalStatus": "DeRegistered",
+  "approvalStatus": "Deregistered",
```

Comment:
Please don't do these kind of changes without asking first. BTA team is already using this API and this would need them changing their code - which is something we can't ask from them considering the timelines. Please revert it back to what it was all throught the code

Reply:
Restored